src/main/java/com/shevtsov/domain/Product.java

```diff
@@ -7,4 +7,41 @@
     private String name;
     private BigDecimal price;
 
-}
+    public Product(String name, BigDecimal price) {
```

Comment:
I do not think that price can be big like BigDecimal

Reply:
Artem defined so. Maybe he has reasons for the future.